app/controllers/posts_controller.rb

```diff
@@ -2,11 +2,34 @@ class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts.includes(:comments)
-    # puts @posts
   end
 
   def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+    @user = User.find(params[:user_id])
+    @current_user = current_user
+  end
+
+  def post_params
+    params.require(:post).permit(:title, :text)
+  end
```

Comment:
[OPTIONAL] Your strong parameters should be in a private method so that external parties wont have access to them

Reply:
thanks so much @fobadara. I would do that